pkg/containerd/cmd/service/skanky-vendor.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+#
+# We only need the containerd client and its transitive dependencies
+# and we conveniently have a checkout already. We actually prefer to
+# reuse containerd's vendoring for consistency anyway.
+
+set -eu
+ctrd=$1
+mkdir -p vendor/github.com/containerd/containerd
+ln -nfs $ctrd/* vendor/github.com/containerd/containerd
+rm -f vendor/github.com/containerd/containerd/vendor
+
+for p in github.com/Microsoft/go-winio \
```

Comment:
Why can't we just copy all of `containerd`s `vendor` directory as we have a checkout? Having more things does not matter.